examples/rh-sso/client-registrations/webhook_handler.lua

```diff
@@ -0,0 +1,105 @@
+local redis_pool = require 'client-registrations/redis_pool'
+local lom = require 'lxp.lom'
+local xpath = require 'luaxpath'
+local cjson = require 'cjson'
+
+local _M = {}
+
+local initial_access_token = "CHANGE_ME_INITIAL_ACCESS_TOKEN"
```

Comment:
Yep. For some reason I thought this is standalone configuration. 

APIcast should support customisation for adding more env variables. Open an issue?